streaming/src/test/scala/org/apache/spark/streaming/ReceiverSuite.scala

```diff
@@ -132,7 +132,7 @@ class ReceiverSuite extends TestSuiteBase with Timeouts with Serializable {
   test("block generator") {
     val blockGeneratorListener = new FakeBlockGeneratorListener
     val blockInterval = 200
```

Comment:
Same, I suppose you could consider renaming this to have "MS" on the end for consistency. The self-documenting name is great.